apps/ledger-live-mobile/src/screens/DebugStoryly.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Platform, ScrollView } from "react-native";
+import { Flex, Text } from "@ledgerhq/native-ui";
+import styled from "styled-components/native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import Alert from "../components/Alert";
+import {
+  storyInstancesIDsMap,
+  StorylyInstanceID,
+} from "../components/Storyly/shared";
+import StoryBar from "../components/Storyly/StoryBar";
+import LanguageSettingsRow from "./Settings/General/LanguageRow";
+
+const StyledStoryBar = styled(StoryBar).attrs({
+  scrollContainerStyle: {
+    paddingHorizontal: 16,
+    justifyContent: "center",
+    flexGrow: 1,
+  },
+})`
+  flex: 1;
+  padding-vertical: 16px;
+`;
+
+const DebugStoryly = () => (
+  <SafeAreaView>
+    <ScrollView>
+      <Flex backgroundColor="background" flex={1}>
```

Comment:
Isn't it `backgroundColor="background.main"`

Reply:
yup, thanks!